Description: Unneccessary shebang line in scripts/vouchers.pl
 Correct issue with unneccessary shebang line in scripts/vouchers.pl
Bug: http://sourceforge.net/tracker/?func=detail&aid=3473848&group_id=175965&atid=875350
Author: Robert James Clay <jame@rocasa.us>

--- a/scripts/vouchers.pl
+++ b/scripts/vouchers.pl
@@ -11,9 +11,6 @@
 
 =cut
 
-#!/usr/bin/perl
-
-
 package LedgerSMB::Scripts::vouchers;
 our $VERSION = '0.1';
 
